src/applications/caregivers/definitions/sharedSchema.js

```diff
@@ -0,0 +1,32 @@
+import { merge, omit } from 'lodash';
+import {
+  yesNoSchema,
+  addressNoMilitarySchema,
+  fullNameSchema as platformFullNameSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+// declare schema for use with non-autofill address definitions
+export const addressSchema = definition =>
+  merge(
+    {},
+    addressNoMilitarySchema({ omit: ['street3'] }),
+    omit(definition, ['additionalProperties', 'required']),
+  );
+
+// declare schema for use with autofill address definitions
+export const addressWithAutofillSchema = definition =>
```

Comment:
Take the standard address schema and add the boolean field for the autofill checkbox.